lib/vdsm/storage/resourceManager.py

```diff
@@ -458,8 +458,8 @@ def registerResource(self, namespace, name, lockType, callback):
             try:
                 namespaceObj = self._namespaces[namespace]
             except KeyError:
-                raise ValueError("Namespace '%s' is not registered with this "
-                                 "manager" % namespace)
+                raise ValueError(f"Namespace '{namespace}' is not registered "
+                                 "with this manager")
```

Comment:
Same for this and other changes in this file.

And of course fix the commit message to reflect reality.